crystal/distribution.yaml

```diff
@@ -948,6 +948,21 @@ repositories:
       url: https://github.com/stonier/py_trees_msgs.git
       version: release/0.4.x
     status: developed
+  py_trees_ros_interfaces:
+    doc:
+      type: git
+      url: https://github.com/splintered-reality/py_trees_ros_interfaces.git
+      version: release/1.0.x
+    release:
+      tags:
+        release: release/crystal/{package}/{version}
+      url: https://github.com/stonier/py_trees_ros_interfaces-release.git
```

Comment:
Do you want to move the release repository to the `splintered-reality` organization too?

Reply:
I'll give another 24h for this question to be addressed before merging the PR as-is if there's no response.

Reply:
No, the release repo remains in the same place.